src/rpc/net.cpp

```diff
@@ -1103,6 +1103,7 @@ UniValue AddrmanEntryToJSON(const AddrInfo& info)
     ret.pushKV("network", GetNetworkName(info.GetNetClass()));
     ret.pushKV("source", info.source.ToStringAddr());
     ret.pushKV("source_network", GetNetworkName(info.source.GetNetClass()));
+    ret.pushKV("mapped_as", info.mapped_as);
```

Comment:
Is it possible to get the AS of the source too? Ideally both?

Reply:
e.g. similar to:

```suggestion
    ret.pushKV("mapped_as", info.mapped_as);
    ret.pushKV("source_mapped_as", info.source_mapped_as);
```

Reply:
Yes!